openlibrary/plugins/openlibrary/js/subjects.js

```diff
@@ -12,9 +12,9 @@
 //          });
 //
 
-;(function() {
+(function() {
```

Comment:
see comments above

Reply:
okay, I have added `;` back and updated the PR.